tox.ini

```diff
@@ -14,7 +14,6 @@ passenv = *
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 deps =
-    -rrequirements.txt
     -rtest-requirements.txt
     ansible24: ansible>=2.4,<2.5
```

Comment:
Oh, mind removing this too, while we're here :smile:

Reply:
Ah, can also do it as part of https://github.com/ansible/molecule/issues/1896 (which was where I orginally caught it), if needed.

Reply:
Yeah, maybe. I'll need to stabilize CI first